CONTRIBUTING.md

```diff
@@ -62,17 +62,10 @@ In addition:
 - Code shall be Python 3.8 + compliant
 
 
-### Use flake8 and/or pylint to check
+### Linting
 
 ```sh
-  python -m flake8 mycode.py
-```
-
-The pylint is rather strict and sometimes exceptions are needed.
-A targeted ``.pylintrc`` file is in the project root.
-
-```sh
-  python -m pylint mycode.py
```

Comment:
I actually think it's fine to keep `pylint`. It doesn't take a big role in the CI cycle because it's extremely pedantic but still checks a great number of things that ruff does not. A good code barometer but not a bar to meet.

However, I don't think we've been using it, so no problem to remove.